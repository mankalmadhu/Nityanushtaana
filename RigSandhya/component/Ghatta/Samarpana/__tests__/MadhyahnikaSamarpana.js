import React from 'react';
import ಮಾಧ್ಯಾಹ್ನಿಕಸಮರ್ಪಣ from '../MadhyahnikaSamarpana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');

test("ಪ್ರಾತಸ್ಸಮರ್ಪಣ", () => {
   expect(Renderer.create(
    <ಮಾಧ್ಯಾಹ್ನಿಕಸಮರ್ಪಣ />
  ).toJSON()).toMatchSnapshot();

});