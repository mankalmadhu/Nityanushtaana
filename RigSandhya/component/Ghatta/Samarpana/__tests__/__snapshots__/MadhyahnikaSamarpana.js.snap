// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತಸ್ಸಮರ್ಪಣ 1`] = `
<View>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    ಸಮರ್ಪಣೆ
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಅನೇನ ಮಾಧ್ಯಾಹ್ನಿಕ ಸ೦ಧ್ಯಾವ೦ದನೇನ  ಭಗವಾನ್ ಸರ್ವಾತ್ಮಕಃ ಶ್ರೀ ತೇಜೋ ಲಕ್ಷ್ಮೀನೃಸಿ೦ಹ ಸ್ವಾಮಿನಃ ಪ್ರೀಣಾತು |
  </ವಿಷಯ>
</View>
`;
